MidCamp/midcamp
# Change field_date to range, update views

To test
- build site
- Add some events to /admin/structure/taxonomy/manage/event including sub-events
- Visit /admin/structure/views/view/event_sub_events and observe the events show up when you enter the parent vid

Todo
- [x] change date only to date and time
- [x] add bool field to hide time
